### PR TITLE
Update docs with go_backwards option

### PR DESCRIPTION
**Pull request by dbonadiman**

### CODE REVIEW

File: docs/sources/layers/recurrent.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Fully connected RNN where output is to fed back to input.
     - __return_sequences__: Boolean. Whether to return the last output in the output sequence, or the full sequence.
     - __input_dim__: dimensionality of the input (integer). This argument (or alternatively, the keyword argument `input_shape`) is required when using this layer as the first layer in a model.
     - __input_length__: Length of input sequences, when it is constant. This argument is required if you are going to connect `Flatten` then `Dense` layers upstream (without it, the shape of the dense outputs cannot be computed).
+    - __go_backwards__: Boolean (Default False).  Process the input sequence backwards.
```
Comment (fchollet) [inline docs/sources/layers/recurrent.md]: No double spacing please.

**Comment by fchollet** [comment]
Cool, thanks!

**Comment by fchollet** [comment]
By the way, remind me, does this feature have tests?

**Comment by dbonadiman** [comment]
The feature does not have test at the moment, i can look in to it if it is required ;)

**Pull request merged**